src/signals/incident/components/form/CategorySelect/CategorySelect.js

```diff
@@ -1,19 +1,42 @@
-import React, { useMemo, useCallback } from 'react';
+import React, { useMemo, useCallback, useState, useEffect } from 'react';
+import styled from 'styled-components';
 import PropTypes from 'prop-types';
 
 import Select from 'components/SelectInput';
 import { useSelector } from 'react-redux';
 import { makeSelectSubCategories } from 'models/categories/selectors';
+import InfoText from 'components/InfoText';
+
+const Wrapper = styled.div`
+  width: 100%;
+  margin-bottom: 0;
+  p {
+    margin-bottom: 0;
+  }
+`;
 
 const CategorySelect = ({ handler, meta, parent }) => {
   const subcategories = useSelector(makeSelectSubCategories);
   const options = useMemo(() => subcategories?.map(({ slug, name }) => ({ key: slug, name, value: slug })), [
     subcategories,
   ]);
+  const { value } = handler();
+
+  const [info, setInfo] = useState();
+
+  const getSubcategory = useCallback(slug => subcategories?.find(
+    s => s.slug === slug
+  ) || {}, [subcategories]);
+
+  useEffect(() => {
+    const { description } = getSubcategory(value);
```

Comment:
What if `getSubcategory` returns `undefined`?

Reply:
It shouldn't. But I added extra tests.